packages/api/internal/handlers/sandbox_get.go

```diff
@@ -0,0 +1,70 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+
+	"github.com/e2b-dev/infra/packages/api/internal/api"
+	"github.com/e2b-dev/infra/packages/api/internal/auth"
+	authcache "github.com/e2b-dev/infra/packages/api/internal/cache/auth"
+	"github.com/e2b-dev/infra/packages/shared/pkg/models/envbuild"
+	"github.com/e2b-dev/infra/packages/shared/pkg/telemetry"
+)
+
+func (a *APIStore) GetSandboxesSandboxID(c *gin.Context, id string) {
+	ctx := c.Request.Context()
+
+	teamInfo := c.Value(auth.TeamContextKey).(authcache.AuthTeamInfo)
+	team := teamInfo.Team
+
+	telemetry.ReportEvent(ctx, "get running instance")
+
+	info, err := a.instanceCache.GetInstance(id)
+	if err != nil {
+		c.JSON(http.StatusNotFound, nil)
```

Comment:
Coudl you add some text explaining the error? 🙏🏻

Reply:
Yes!